### ISSUE TITLE
Pressing TAB while editor's focused changes the focus of the UI instead of doing a tabulation

### ISSUE THREAD

**Issue by MLefebvreICO** [microsoft/vscode#88707]
- VSCode Version: 1.41.1
- OS Version: Windows 10 Pro 64bits

Steps to Reproduce:

Steps currently unknown ..

Sometimes when coding, I'll press TAB key to insert a new tabulation while I have the editor focused, however, it will only focus on the next UI focusable option (`Problems` pane here ..).

Attaching a gif of the issue : 
![VSCodeTabFocusIssue](https://user-images.githubusercontent.com/17457252/72465123-9740ec00-37a4-11ea-83a3-734dfc98d111.gif)

Only fix is to restart VSCode until it will happen again in the future ..

Does this issue occur when all extensions are disabled?: Unknown

**Comment by IllusionMH** [comment]
/needsMoreInfo

Do you have "Tab moves focus" in the status bar?
See details here https://code.visualstudio.com/docs/editor/accessibility#_tab-trapping

**Comment by MLefebvreICO** [comment]
@IllusionMH I haven't notice and have restarted since.
Just checking that now, it's doing the same thing as I was observing, however the `Tab moves focus` is not red on my end, so it's not easy to see.

![image](https://user-images.githubusercontent.com/17457252/72528253-9f4a6b80-3838-11ea-80ef-84dc048f982b.png)


Maybe consider putting in back to red or maybe change the default shortcut to something people won't trigger by accident ?

**Comment by IllusionMH** [comment]
Not sure if red background will be good - too much distraction to those who actually use them, but might be fair request if it's possible to easily override that color.

But lets wait for someone from VS Code team to weight in on this.

**Comment by MLefebvreICO** [comment]
Just to throw another idea on the table : maybe it should show a warning popup (or a notification ??) like Windows does with the sticky keys ?

The way I see, there's a minimum of 3 ways to make this more obvious : 

1. Tell the user what he's about to do and confirm for his choice (with a checkbox to remember the choice or not)
2. Only show a popup notification of some kind to inform the user he has toggled it.
3. Do the bare minimum and simply put this back with a red background so we can see it, because right now, you really have to know it's there ...

**Comment by alexdima** [comment]
@MLefebvreICO We want VS Code to be accessible to all and this mode is an important part of making VS Code accessible to keyboard users. We will not show popups or anything that makes VS Code annoying to users who must use this mode.

The tab moves focus mode is entered when executing the action "Toggle Tab Key Moves Focus". This action is bound by default to `Ctrl+M` on Windows. 
 
Please simply write the following to your `keybindings.json`:
```json
{ "key": "ctrl+m", "command": "-editor.action.toggleTabFocusMode" }
```

**Comment by MLefebvreICO** [comment]
@alexdima I understand that people uses it and that it not should be annoying for them, but why not put a simple red background to make it pop a little more like in the docs.

I have changed my shortcut on my end, but I find it quite confusing to not show more for people that types rapidly and hit `M` instead of `N` for a new file, let's say.

**Comment by alexdima** [comment]
The color was removed in https://github.com/microsoft/vscode/commit/6e94782f89437e3f22e9f0bfc810a0090d71f7ee#diff-18abda3e2f9340124237b5bc04362d11

**Comment by bpasero** [comment]
Since we have an explicit theming color for this purpose (`statusBarItem.prominentBackground`) I am using that now:

![image](https://user-images.githubusercontent.com/900690/72784029-1f514680-3c28-11ea-928a-c5f7470b9685.png)

@alexdima should this color be used also for the "Screen Reader Optimized" entry?

**Comment by alexdima** [comment]
Good point, pushed via https://github.com/microsoft/vscode/commit/3abcebe85021a0f78db9dfbae73edbb57ceadc94
